stereo_click/ClickPoint click_point
---
geometry_msgs/PointStamped pt3d
